src/data-search.js

```diff
@@ -41,7 +41,35 @@ export async function getData () {
     const papers = await searcher.search(keywords, {
       combineWith: 'AND'
     });
-    return _.assign({}, config, { papers });
+
+    // Filter for multiple versions of the same paper
+    const unique = [];
+    for (const paper of papers) {
+      const exists = unique.find((obj) => obj.title === paper.title);
+
+      if (!exists) {
+        const allVersions = papers.filter((obj) => obj.title === paper.title);
+        if (allVersions.length > 1) {
+          const max = allVersions.reduce((maxObj, obj) => {
+            if (obj.version > maxObj.version) {
+              return obj;
+            } else {
+              return maxObj;
+            }
+          }, { version: -Infinity });
+          unique.push(max);
+        } else {
+          unique.push(paper);
+        }
+      }
+    }
+
+    // Find and save final link for each paper's DOI
+    for (const paper of unique) {
+      const finalURL = `https://www.${paper.server}.org/content/${paper.doi}v${paper.version}`;
+      paper.finalURL = finalURL;
+    }
+    return _.assign({}, config, { unique });
```

Comment:
By renaming `papers` to `unique`, the client breaks (because it assumes the existence of the attribute `papers`).